### ISSUE TITLE
ValueError: Did not recognize loaded array layout

### ISSUE THREAD

**Issue by jakeyizle** [Canisback/roleML#16]
After installing roleml through pip, when I try to import roleml I get the following error:

Traceback (most recent call last):
  File "main.py", line 1, in <module>
    from roleml import roleml
  File "C:\Users\Jakertle\AppData\Local\Programs\Python\Python37-32\lib\site-packages\roleml\roleml.py", line 111, in <module>
    roleml_model = joblib.load(os.path.join(os.path.dirname(__file__), "role_identification_model_32bits.sav"))
  File "C:\Users\Jakertle\AppData\Local\Programs\Python\Python37-32\lib\site-packages\joblib\numpy_pickle.py", line 585, in load
    obj = _unpickle(fobj, filename, mmap_mode)
  File "C:\Users\Jakertle\AppData\Local\Programs\Python\Python37-32\lib\site-packages\joblib\numpy_pickle.py", line 504, in _unpickle
    obj = unpickler.load()
  File "C:\Users\Jakertle\AppData\Local\Programs\Python\Python37-32\lib\pickle.py", line 1085, in load
    dispatch[key[0]](self)
  File "C:\Users\Jakertle\AppData\Local\Programs\Python\Python37-32\lib\site-packages\joblib\numpy_pickle.py", line 329, in load_build
    Unpickler.load_build(self)
  File "C:\Users\Jakertle\AppData\Local\Programs\Python\Python37-32\lib\pickle.py", line 1549, in load_build
    setstate(state)
  File "sklearn\tree\_tree.pyx", line 673, in sklearn.tree._tree.Tree.__setstate__
ValueError: Did not recognise loaded array layout

**Comment by Canisback** [comment]
It seems like I'll never make it work for 32 bits.
Can you give me the versions installed for scikit-learn and numpy?
If scikit-learn is not 0.23.0, try to force it to this version and see if it works.

Also I recommend using python 64 bits to avoid a lot of these problems.

**Comment by jakeyizle** [comment]
I just installed 64 bit python and it is indeed working! Unfortunately I uninstalled my 32 bit at the same time so I'm not sure which versions of those packages I had, but I did try forcing scikit to 0.22.1 with no success.

I am now getting a different error, but I assume it's the way I'm passing match data in to the predict function.


    if match["gameDuration"] < 720:
TypeError: byte indices must be integers or slices, not str
